meetings/2023-10-18.md

```diff
@@ -63,23 +102,57 @@ URL: https://lists.w3.org/Archives/Public/public-solid/2023Oct/0036.html
 * SC: https://github.com/solid/authorization-panel/pull/331
 * SC: https://github.com/solid/webid-profile/pull/108
 * SC: https://github.com/solid/type-indexes/pull/28
-* SC: There are a few other documents that I didn't get to checking. Any objections?
+* SC: There are a few other documents that I didn't get to checking. Please review or Any objections or merge?
 * SC: This is also mentioned in https://github.com/solid/specification/blob/main/meetings/2023-10-11.md#wg-charter-proposal-status-update . See related https://github.com/solid/solid-wg-charter/pull/58 .
-
-
-### extend timeline
+* SC: I will leave them for a while. The CG does not use the Solid process any more. The focus of the Solid Process on the Director raised some issues with the AC review of the WG charter proposal.
+* eP: I'm just noticing that the WG charter proposal points to solidproject.org/TR/ . This needs to be updated as well.
+* SC: There is also an issue of using CG templates. WG charter could be updated to refer to EDs alternatively.
+* PAC: As I wrote in 58 "goes without saying" - the WG will follow the W3C Process and not other process. I'm not concerned about what those pointed documents say. They are just some input. The WG will create a new draft under the W3C Process based on that content. 
+* SC: Is it sufficient if we are updating the EDs rather than the ones in TRs?
+* PAC: There will be some text accompanying. In this text I can emphasis that the CG is using community-governance per CG charter. This move is occurring even if it is in the EDs, it is enough evidence for me.
+* SC: any objection to merging these PRs? All they do is remove the mention to the solid process?
+* eP: this is editorial change, so we can change only the third (patch) part of the version. Also, "editor's draft" is only for Working/Interest Groups. We should be using the CG templates.
+* SC: I've created an issue for that somewhere in w3c. They were supposed to update the process.
+* PAC: CG template mentions ED, e.g., https://www.w3.org/2021/12/rdf-star.html . Even if only mentioned for WGs, it wasn't the intention reserve that term.
+
+
+### WG Charter PRs from Reviews
+* SC: We are over time, but the following PRs on the WG charter are mostly editorial. I would like to get rid of them. I'll assign them to PAC, and he can merge them if there are positive reviews after a week since their creation.
+* SC: https://github.com/w3c/charter-drafts/issues/461 relates to https://github.com/w3c/charter-drafts/issues/459
+* PAC: thanks for bringing that up to the more general level. However, regardless of whether this is integrated in the template or not, I would prefer to keep it in our charter. Every charter discussion is different and justifies adjustments to the template. As for "It goes without saying", it could be counter-productive, but I want to emphasize the fact that we do not consider this as a "gift" that we are doing -- we know that this is how things are supposed to be.
```

Comment:
```suggestion
* PAC: Thanks for bringing that up to the more general level. Regardless of whether or not this is integrated in the template, I would prefer to keep it in our charter. Every charter discussion is different and justifies adjustments to the template. As for, "It goes without saying", it could be counter-productive, but I want to emphasize the fact that we do not consider this as a "gift" that we are doing — we know that this is how things are supposed to be.
```